src/content/en/2020/markup.md

```diff
@@ -491,47 +553,49 @@ The most popular ones stand out because they are almost twice as popular than ea
 <figcaption>{{ figure_link(caption="The most popular <code>data-*</code> attributes.", sheets_gid="764700773", sql_file="pages_almanac_by_device_and_data_attribute_name_frequency.sql") }}</figcaption>
 </figure>
 
-Attributes like `data-type`, `data-id`, and `data-src` can have multiple generic uses although `data-src` is used a lot with lazy image loading via JavaScript (e.g., Bootstrap 4). [Bootstrap](https://getbootstrap.com/) again explains the presence of `data-toggle`, where it’s used as a state styling hook on toggle buttons. The [Slick carousel plugin](https://kenwheeler.github.io/slick/) is the source of `data-slick-index`, whereas `data-element_type` is part of [Elementor’s WordPress website builder](https://elementor.com/). Both `data-requiremodule` and `data-requirecontext`, then, are part of [RequireJS](https://requirejs.org/).
+Attributes like `data-type`, `data-id`, and `data-src` can have multiple generic uses although `data-src` is used a lot with lazy image loading via JavaScript (e.g., Bootstrap 4). [Bootstrap](https://getbootstrap.com/) again explains the presence of `data-toggle`, where it's used as a state styling hook on toggle buttons. The [Slick carousel plugin](https://kenwheeler.github.io/slick/) is the source of `data-slick-index`, whereas `data-element_type` is part of [Elementor's WordPress website builder](https://elementor.com/). Both `data-requiremodule` and `data-requirecontext`, then, are part of [RequireJS](https://requirejs.org/).
 
-<p class="note">Interestingly, the use of native lazy loading on images is similar to that of <code>data-src</code>. 3.86% of <code>img</code> elements use the <code>loading</code> attribute with a value of <code>lazy</code> (this appears to be growing very fast, as <a href="https://twitter.com/zcorpan/status/1237016679667970050">back in February, this number was about 0.8%</a>). It’s possible that these are being used together for a <a href="https://addyosmani.com/blog/lazy-loading/">cross-browser solution</a>.</p>
+Interestingly, the use of native lazy loading on images is similar to that of `data-src`. 3.86% of `img` elements use the `loading` attribute with a value of `lazy`. This appears to be growing very fast, as back in February, this number was about [0.8%](https://twitter.com/zcorpan/status/1237016679667970050). It's possible that these are being used together for a [cross-browser solution](https://addyosmani.com/blog/lazy-loading/).
```

Comment:
Hmm, I thought the 3.86% number was of pages using `<img loading=lazy>`, not of `img` elements. If it's not % of pages, the numbers are not comparable.

Reply:
cc @Tiggerito

Reply:
@zcorpan is right. This data is on a per page bases. So pages that have an img with loading set to lazy.

Reply:
So the "this appears to be growing fast..." sentence should be removed?

Reply:
I've corrected the phrasing of the stat itself but left a TODO for the authors to update their interpretation.

Reply:
@iandevlin no it can stay, since both stats are of pages.